Dockerfile

```
@@ -150,6 +150,9 @@ ENV ALLOWED_HOSTS '["*"]'
 # Copy the application code over
 COPY cfgov ./cfgov/
 COPY static.in ./static.in/
+COPY refresh-data.sh .
```

This file is missing

```suggestion
COPY refresh-data.sh .
COPY initial-data.sh .
```

---

I added this as part of my suggestions in #8709, which has now been merged into this PR.